Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/sparkserverless/SparkServerlessClusterRootModule.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ *
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ * the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.microsoft.tooling.msservices.serviceexplorer.azure.sparkserverless;
+
+import com.google.common.collect.ImmutableList;
+import com.microsoft.azuretools.azurecommons.helpers.AzureCmdException;
+import com.microsoft.azuretools.azurecommons.helpers.NotNull;
+import com.microsoft.tooling.msservices.serviceexplorer.Node;
+import com.microsoft.tooling.msservices.serviceexplorer.AzureRefreshableNode;
+
+import java.util.List;
+
+public class SparkServerlessClusterRootModule extends AzureRefreshableNode{
+    private static final String SparkServerless_SERVICE_MODULE_ID = SparkServerlessClusterRootModule.class.getName();
+    // TODO: Update icon path
+    private static final String ICON_PATH = "StorageAccount_16.png";
```

Comment:
Could you provide a new icons list for designer?

Reply:
I'm not sure where to find new icons. Could you give a hand with the icons?

Reply:
I can find one for you

---
In reply to: [186345959](https://github.com/Microsoft/azure-tools-for-java/pull/1625#discussion_r186345959) [](ancestors = 186345959)